virtualizor.py

```diff
@@ -104,6 +104,17 @@ def __init__(self, conf):
         self.target_host = conf.target_host
         self.conn = libvirt.open('qemu+ssh://root@%s/system' %
                                  conf.target_host)
+        self.emulator = self.find_emulator()
+        if self.emulator is None:
+            logging.error("No emulator found")
+            exit(2)
+
+    def find_emulator(self):
+        for location in ('/usr/bin/qemu-system-x86_64',
+                         '/usr/libexec/qemu-kvm'):
+            if self.call('test', '-f', location) == 0:
+                return location
+        return None
```

Comment:
I think you should replace the "return None" by:

````
raise Exception("No emulator found")
````

This way, you can simplify the code. BTW, this error is unlike to happen a lot.